comp/core/health-platform/README.md

```diff
@@ -0,0 +1,180 @@
+# Health Platform Component
+
+The Health Platform component provides a mechanism for collecting and reporting health information from the Datadog Agent to the backend. It includes hostname information, host ID from gopsutil, organization ID, and a list of issues with their details.
```

Comment:
Should you also add something along the following line "... to show customer in Datadog UI that that particular Agent is healthy and instructions to address it or notify user by other means"